lib/maths/CTimeSeriesModel.cc

```diff
@@ -99,6 +99,7 @@ double aggregateFeatureProbabilities(const TDouble4Vec& probabilities, double co
 const std::string VERSION_6_3_TAG("6.3");
 const std::string VERSION_6_5_TAG("6.5");
 const std::string VERSION_7_3_TAG("7.3");
+const std::string VERSION_7_11_TAG("7.11");
```

Comment:
One more thing to consider from a backport perspective is the naming of this tag.  It's going to look a bit weird to have "VERSION_7_11_TAG" in 6.8.14.  So maybe:

```suggestion
const std::string VERSION_7_11_AND_6_8_14_TAG("7.11and6.8.14");
```

Reply:
I think my inclination is just to rename this in 6.8 since I'll need to make changes to get this to merge anyway and we shouldn't get conflicts here. I'll get a review on that PR.